.github/workflows/jobs-create-env-mock-data.yml

```diff
@@ -1,3 +1,10 @@
+#
+# Create mock data workflow
```

Comment:
I don't think this is really creating 'mock' data (and is probably beyond the scope of this PR - so pls ignore)   This script merely creates the Participants to allow the system to function with the deployed components.